CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Use Dataservice search [#565](https://github.com/datagouv/udata-front/pull/565)
 - Add organization type filter to dataset search [#579](https://github.com/datagouv/udata-front/pull/579)
 - Fix Reuse tests [#580](https://github.com/datagouv/udata-front/pull/580)
+- Add API menu item [#578](https://github.com/datagouv/udata-front/pull/578)
```

Comment:
This section should be in current. And it can mention publishing button as well :)